### ISSUE TITLE
[BUG] Non-Existent Control -1

### ISSUE THREAD

**Issue by drinfernoo** [jurialmunkey/skin.arctic.horizon.2#641]
### Skin section

Other

### Current Behavior

I am seeing hundreds of `EXCEPTION: Non-Existent Control -1` messages in my log while using this skin... At first I thought it might've only been happening inside of addon-defined windows, but I am also getting it with nothing installed but the skin.

They happen pretty much constantly, with this type of thing showing up pretty regularly:
```
2022-11-14 18:04:01.342 T:32936    INFO <general>: Loading skin file: SettingsCategory.xml, load type: KEEP_IN_MEMORY
2022-11-14 18:04:01.361 T:1892    ERROR <general>: EXCEPTION: Non-Existent Control -1
2022-11-14 18:04:10.939 T:32936    INFO <general>: Skipped 45 duplicate messages..
2022-11-14 18:04:10.939 T:32936    INFO <general>: Loading skin file: SkinSettings.xml, load type: LOAD_EVERY_TIME
2022-11-14 18:04:10.968 T:1892    ERROR <general>: EXCEPTION: Non-Existent Control -1
2022-11-14 18:04:13.342 T:19340    INFO <general>: Skipped 11 duplicate messages..
```
And it seems to happen in pretty much any window 🤔 

### Expected Behavior

I would think this is an indication of _some_ type of bug... but at the least it is quite annoying 😫

Can we figure out where this is coming from?

### Steps To Reproduce

1. Install Arctic Horizon 2.
2. Navigate to Skin Settings.

### Screenshots and Additional Info

https://hastebin.com/ecuroboxom

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Fixed in v5.0.24 of TMDbHelper.

**Comment by drinfernoo** [comment]
Thank you!
